patches/server/0001-Setup-Gradle-project.patch

```diff
@@ -98,19 +98,23 @@ index 0000000000000000000000000000000000000000..f99a9702fe9282a1982c25cd0c003d8d
 +    }
 +}
 +
++val noRelocation = providers.systemProperty("Paper.no-relocation").forUseAtConfigurationTime();
```

Comment:
I think I'd prefer this to just be a separate task. Would be simpler, 2 separate tasks that do different things, rather than 1 task which will do different things depending on the environment. The common parts of the task configuration could be pulled out via extension functions (examples):

https://github.com/PaperMC/paperweight/blob/main/buildSrc/src/main/kotlin/config-publish.gradle.kts#L14-L24
https://github.com/PaperMC/paperweight/blob/main/buildSrc/src/main/kotlin/config-publish.gradle.kts#L29

&

https://github.com/PaperMC/paperweight/blob/main/buildSrc/src/main/kotlin/config-publish.gradle.kts#L111-L138
https://github.com/PaperMC/paperweight/blob/main/buildSrc/src/main/kotlin/config-publish.gradle.kts#L67-L84